src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function operate(callback, number) {
+    result = callback(result, number);
+
+    return this;
+  }
+
+  function reset() {
+    result = 0;
+
+    return this;
```

Comment:
The `reset` method should return the calculator object itself to allow method chaining, but currently, it returns `this` which refers to the global object since `reset` is not an arrow function and is not bound to the calculator object.